src/porepy/examples/flow_benchmark_3d_case_3.py

```diff
@@ -87,27 +85,14 @@ def intersection_permeability(self, subdomains: list[pp.Grid]) -> pp.ad.Operator
         # Use `fracture_permeability` as intersection permeability under the assumption
         # that they are equal. This is valid in the current benchmark case.
         permeability = pp.wrap_as_dense_ad_array(
-            self.solid.fracture_permeability(), size, name="intersection permeability"
+            self.solid.fracture_permeability, size, name="intersection permeability"
         )
         return self.isotropic_second_order_tensor(subdomains, permeability)
 
 
-class BoundaryConditions:
+class BoundaryConditions(pp.BoundaryConditionMixin):
```

Comment:
Yes I think it would achieve the same.
This is related to https://github.com/pmgbergen/porepy/pull/1252#discussion_r1822219015
Should we make the general change for all the available mixins, providing examples for BC, grids, etc.?

Reply:
Yes, I support that. Should we do it as a separate PR, though?

Reply:
Yes, I think another issue should document this.

Reply:
Has the issue been made? When so, feel free to resolve.

Reply:
Created #1260